<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRelacion</class>
 <widget class="QWidget" name="FormRelacion">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reglas de relaci√≥n</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>CADA </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblInicioA">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string notr="true">ENTIDAD-A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="cbCardinalidadInicio"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leLabelInicio"/>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="cbMultiplicidadFinal"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="lblFinalA">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string notr="true">ENTIDAD-B</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>CADA </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lblFinalB">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string notr="true">ENTIDAD-B</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="cbCardinalidadFinal"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leLabelFinal"/>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="cbMultiplicidadInicio"/>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="lblInicioB">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string notr="true">ENTIDAD-A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
